framework/src/modules/poa/commands/update_authority.ts

```diff
@@ -54,50 +53,87 @@ export class UpdateAuthorityCommand extends BaseCommand {
 		context: CommandVerifyContext<UpdateAuthorityParams>,
 	): Promise<VerificationResult> {
 		const { newValidators, threshold, validatorsUpdateNonce } = context.params;
-
-		if (newValidators.length < 1 || newValidators.length > MAX_NUM_VALIDATORS) {
-			throw new Error(
-				`newValidators length must be between 1 and ${MAX_NUM_VALIDATORS} (inclusive).`,
-			);
+		try {
+			validator.validate(updateAuthoritySchema, context.params);
```

Comment:
I think the schema validity is checked separately in the state machine itself, see [here](https://github.com/LiskHQ/lisk-sdk/pull/8558/files#diff-cb2314f09fb36abaa9b56f7e516e6753a48441c7e8ac30478fd6ad2a9f998360).